plugins/services/src/js/components/forms/ContainerServiceFormAdvancedSection.tsx

```diff
@@ -238,7 +240,9 @@ class ContainerServiceFormAdvancedSection extends React.Component {
     const artifactErrors =
       findNestedPropertyInObject(errors, artifactsPath) || [];
     const diskPath = this.getFieldPath(path, "disk");
+    const limitsPath = "limits";
     const diskErrors = findNestedPropertyInObject(errors, diskPath);
+    const limitsErrors = findNestedPropertyInObject(errors, limitsPath);
```

Comment:
lets prefer type safe helpers (e.g. `error?.limits` in this case) over `findNestedPropertyInObject`.

Reply:
✅